e2e/etcdctlv3_test.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/coreos/etcd/pkg/fileutil"
+	"github.com/coreos/etcd/pkg/testutil"
+)
+
+func TestCtlV3SetQuorum(t *testing.T) {
+	checkTesting(t, testCtlV3Set(t, &configNoTLS, 5*time.Second, true))
+}
+
+func TestCtlV3SetQuorumTimeout(t *testing.T) {
+	err := testCtlV3Set(t, &configNoTLS, time.Nanosecond, true)
+	if !strings.Contains(err.Error(), "grpc: timed out trying to connect") {
+		t.Fatalf("expected timeout error, got %v", err)
+	}
+}
+
+// TODO: Getting transport: x509: certificate signed by unknown authority
+// func TestCtlV3SetClientTLSQuorum(t *testing.T) {
+// 	checkTesting(t, testCtlV3Set(t, &configClientTLS, 5*time.Second, true))
+// }
+
+func TestCtlV3SetPeerTLSQuorum(t *testing.T) {
+	checkTesting(t, testCtlV3Set(t, &configPeerTLS, 5*time.Second, true))
+}
+
+// TODO: Getting transport: x509: certificate signed by unknown authority
+// func TestCtlV3SetTLSQuorum(t *testing.T) {
+// 	checkTesting(t, testCtlV3Set(t, &configTLS, 5*time.Second, true))
+// }
+
+func checkTesting(t *testing.T, err error) {
```

Comment:
I had it for testing timeout. Should I just add another arg to check the timeout error?